Src/WitsmlExplorer.Api/Startup.cs

```diff
@@ -57,6 +61,48 @@ public void ConfigureServices(IServiceCollection services)
             services.ConfigureDependencies(Configuration);
             services.AddHostedService<BackgroundWorkerService>();
             services.AddScoped<ICopyLogDataWorker, CopyLogDataWorker>();
+            services.AddEndpointsApiExplorer();
+            services.AddSwaggerGen(options =>
+            {
+                var basicSecurityScheme = new OpenApiSecurityScheme
```

Comment:
Maybe move out this code block to its own file a la "SetupSwagger" to keep Startup more concise?